supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesResponseJsonFormatter.scala

```diff
@@ -2,24 +2,19 @@ package cromwell.backend.impl.tes
 
 import spray.json._
 
-final case class TesPostResponse(value: String)
-
-final case class TesGetResponse(
-                                 jobID: String,
-                                 task: TesTaskMessage,
-                                 state: String,
-                                 logs: Option[Seq[JobLogs]]
-                               )
+final case class CreateTaskResponse(id: String)
 
+final case class CancelTaskResponse()
```

Comment:
Astonishingly, this seems to be true - https://github.com/spray/spray-json/issues/186

I'm curious what this serializes to, presumable an empty json object (`{}`)?  Is that what you want?

I'm not too fussed either way but it should be possible to make a `new RootJsonFormat[CancelTaskResponse]` for the case object, where write creates a `JsString` and read would just be a string comparison vs the literal string `"CancelTaskResponse"`.